### DISCUSSION TITLE
When do you expect to release a new version of learnr to CRAN?

### DISCUSSION THREAD

**Discussion by davidkane9** [rstudio/learnr#699] [General]
The [current version](https://cran.r-project.org/web/packages/learnr/index.html) was released more than two years ago. There was [some discussion](https://github.com/rstudio/learnr/discussions/617#discussioncomment-1744653) that a new version might be released six months ago. I would prefer to rely on a CRAN version rather than having my students install from Github.

**Comment by davidkane9** [comment]
@gadenbuie Any updates on this? The version on CRAN is 0.10.1. The version here, on Github, is 0.10.3.

**Reply by gadenbuie** [reply]
As you can imagine after having been clearly very wrong about an imminent release when I answered 6 months ago, I'm reticent to give a time frame. If installation is the primary hurdle, you may be interested in using https://rstudio.r-universe.dev/ui#package:learnr which provides a CRAN-like repository with the latest learnr version that can be installed via `install.packages()`.

**Reply by davidkane9** [reply]
Thanks for the suggestion! Do you have a sense of the reliability of the rstudio.r-universe.dev site? The reason I ask is that I have lots of students, often during class, install learnr. Do the extent that rstudio.r-universe.dev is not reliable, I might be better off sticking with Github.

**Reply by gadenbuie** [reply]
I think it's very reliable, or at least as reliable as GitHub. The nice part, in addition to allowing package install with `install.packages()`, is that it can provide binaries for Mac and Windows (although currently the Windows builds aren't working so at the moment they install from source).

**Comment by gadenbuie** [comment]
learnr v0.11.0 is now on CRAN! Read all about the changes in [the learnr v0.11.0 release notes](https://pkgs.rstudio.com/learnr/articles/releases/learnr_v0-11-0.html).
